k8s/helmfile/helmfile.yaml

```diff
@@ -163,7 +163,7 @@ releases:
 
   - name: api
     chart: wbstack/api
-    version: "0.31.2"
+    version: '{{- if eq .Environment.Name "production" }} "0.31.2" {{else}} "0.32.0" {{end}}'
```

Comment:
`in ./helmfile.yaml: [improper constraint:  "0.32.0" ]` seems to be what I see if I try to `diff` this to staging. Do you see that?

Reply:
yes I see that.